uiowa/uiowa
# Writing University Serif font additions

PR for work on https://github.com/uiowa/uiowa/issues/2807

# To Test
1. Pull this Branch
2. blt frontend
3. Sync either sandbox.uiowa.edu site or writinguniversity.org. I recommend sandbox.uiowa.edu as it has people.
4. You should observe that an update hook ran.
5. uli in to the site and navigate to Appearance > Settings > uids_base
6. In the Font family dropdown, select Serif(Zilla Slab)
7. Observe that there is Zilla Slab font in these Places:
    1. Articles Page Body field and WYSIWYG 
    2. People Page Biography and WYSIWYG
    3. Events Page Description
    4. Page body Field and WYSIWYG
    5. Layout builder text area and WYSIWYG
